src/Acuminator/Acuminator.Utilities/Roslyn/Semantic/PXContext.cs

```diff
@@ -93,6 +94,10 @@ public class PXContext
 		public PXGraphExtensionSymbols PXGraphExtension => _pxGraphExtensionSymbols.Value;
 
 		public INamedTypeSymbol PXCacheExtensionType => Compilation.GetTypeByMetadataName(TypeFullNames.PXCacheExtension);
+		public INamedTypeSymbol PXGraphExtensionType => Compilation.GetTypeByMetadataName(TypeFullNames.PXGraphExtension);
```

Comment:
We already have PXGraphExtension type just few lines above. Please remove this duplicate. 

As for the generic types - they should be removed too. We can't use just a couple of generic types, we must support all generic graph extension types, there could be high level chained graph extensions that have much more type parameters. It's not an option to hard code the types, so they should be also removed.

Reply:
Removed.